topics/digitisations.md

```diff
@@ -33,8 +33,8 @@ body_class: digitisations
 | `status` | `VARCHAR(64)` | Request status | `LIVE` |
 | `rolled_over_from` | `INTEGER` | If this request was created as part of a roll over, the identifier of the original request | `1` |
 | `rolled_over_to` | `INTEGER` | If this request was itself rolled over, the identifier of the newly created rolled over request | `3` |
-| `date_copied` | `TIMESTAMP` |  |  |
-| `date_released` | `TIMESTAMP` |  |  |
+| `date_copied` | `TIMESTAMP` | Date that the copy was added to the request | `2017-08-24 00:00:00` |
+| `date_released` | `TIMESTAMP` | The date the copy was made live to the user. Usually request start date or latest bundle creation date | `2017-08-24 00:00:00` |
```

Comment:
This is mostly right. I would change `request start date` to `course start date or the date the request was made available to students, if after the course start date`